feat: add jas1005

jas1005 opened the pull request:
<!-- Please fill in all areas in this PR form. Incomplete PRs will be marked invalid and may be closed. -->

## Description

This PR adds @jas1005 as a contributor.

<!--
Please do not leave this blank
This PR adds <github-username> as a contributor.
-->

## What type of PR is this? (check all applicable)

- [X] 🤝 Add a contributor
- [ ] 📝 Documentation Update

## Related Issues

<!-- 
Please use this format link issue numbers: Closes #123
More information: https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword 
-->

Closes #445 

## Contributors Checklist

### I've read through the [Getting Started](https://intro.opensauced.pizza/#/intro-to-oss/how-to-contribute-to-open-source?id=getting-started) section.

- [X] ✅ Yes
- [ ] ❌ Not yet

### Have you run `npm run contributors:generate` to generate your profile and the badge on the README?

- [X] ✅ Yes
- [ ] ❌ No

## Added to documentation?

- [X] 📜 README.md
- [ ] 🙅 no documentation needed

## Screenshot (Required for PR Review)

<!-- Please provide a screenshot of your profile being generated on the README. This ensures that you ran the `npm run contributors:generate` command, as mentioned in the previous question, which makes it easier for the maintainers to review PRs. All PRs without screenshots will be automatically rejected-->

<img width="118" alt="contributor_profile" src="https://github.com/user-attachments/assets/ac1c4d69-9a0a-4081-a73a-adabc7869d48">

## [optional] What GIF best describes this PR or how it makes you feel?

<!-- note: PRs with deleted sections will be marked invalid -->

<!--
  For Work In Progress Pull Requests, please use the Draft PR feature,
  see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.

  For a timely review/response, please avoid force-pushing additional
  commits if your PR already received reviews or comments.

  Before submitting a Pull Request, please ensure you've done the following:
  - 📖 Read the Open Sauced Contributing Guide: https://github.com/open-sauced/.github/blob/main/CONTRIBUTING.md.
  - 📖 Read the Open Sauced Code of Conduct: https://github.com/open-sauced/.github/blob/main/CODE_OF_CONDUCT.md.
  - 👷‍♀️ Create small PRs. In most cases, this will be possible.
  - 📝 Use descriptive commit messages.
  - 📗 Update any related documentation and include any relevant screenshots.
-->

jas1005 commented:
@open-sauced/docs, please see the comments section of my original pull request. I have updated the title of my pull request to be in accordance with the Open Sauced rules. Please advise me if there is anything else to be done before my code can be merged.

adiati98 commented on contributor_profile.png:
You need to upload this image on your PR template for us to review, and no need to include it in your PR.

You can simply drag and drop the image to the PR template. :)

jas1005 replied:
Thank you for the clarification! I have uploaded the image to the PR template. I will now resolve the merge conflicts and remove my profile confirmation from my commit.

adiati98 commented:
Hey @jas1005,

We have merged a PR before yours. Please **[resolve the conflicts](https://intro.opensauced.pizza/#/intro-to-oss/how-to-contribute-to-open-source?id=merge-conflicts-in-the-guestbook-repository)** before we can review your PR.

Please let me know if you need any help. Thanks. 😊

jas1005 commented:
@adiati98 I think I've fixed all the issues in a way that's in accordance with contribution guidelines. You can see that I put my screenshot into the PR template above and undid my commit where I added a separate file. I had to manually fix the merge issues with me adding myself to the author list and all the other authors that added themselves to the author list between when I forked the repository. I think I've done this correctly though. Can you confirm this? If there's anything else I need to do, I would love your guidance on how to proceed. I spent a large fraction of today going down rabbit holes to figure out where I went wrong.

adiati98 commented on .all-contributorsrc:
```diff
@@ -1,6 +1,6 @@
 {
   "projectName": "opensauced-guestbook",
-  "projectOwner": "TejasQ",
+  "projectOwner": "TeQ",
```
We don't want to change this. Please change it back to the original.

jas1005 replied:
Hey, @adiati98. I don't know why this difference is showing up. When I'm on my main branch or my feature branch, I see the full 'TejasQ' on line 3. Can you advise on how I should locate the version of .all-contributorsrc that is giving this conflict?

adiati98 reviewed (changes requested):
Hi @jas1005,
Thank you for resolving the merge conflicts!

I left a review here for you to fix.

Also, you need to run the command to generate your profile after you merged the conflicts. I don't see the badge increase here. 

Another thing is please provide a description of your PR in the PR template.

Let me know if you have questions or need help! ✨

jas1005 commented:
@adiati98, I think I figured it out. I hadn't pushed/pulled my changes to my fork in the right way. I think I've addressed all of the issues you mentioned in your most recent thread. Is there anything else I should do?

adiati98 reviewed (approved):
Well done and thank you for the fix, @jas1005! ✨ 

If you haven't, you can join our [Community](https://github.com/orgs/open-sauced/discussions/1).
Also, check out our [pizza-verse](https://github.com/open-sauced/pizza-verse) repo if you want to contribute further! 🍕🍴

![git merge GIF](https://media.giphy.com/media/cFkiFMDg3iFoI/giphy.gif)

jas1005 commented:
@adiati98 @BekahHW, am I good to do `git merge` then? How exactly should I do that given my somewhat unique situation with my commits?

adiati98 commented:
> @adiati98 @BekahHW, am I good to do `git merge` then? How exactly should I do that given my somewhat unique situation with my commits?

You don't need to do anything. There will be a 2nd review, and one of the maintainers will merge your PR. 

So, how things go in open source is you create a PR, then the maintainer(s) of the project will review it. They might or might not ask you for fixes, and they will merge the PR after it meets the requirements. Hope this explanation helps. 🙂

jas1005 commented:
> > @adiati98 @BekahHW, am I good to do `git merge` then? How exactly should I do that given my somewhat unique situation with my commits?
> 
> You don't need to do anything. There will be a 2nd review, and one of the maintainers will merge your PR.
> 
> So, how things go in open source is you create a PR, then the maintainer(s) of the project will review it. They might or might not ask you for fixes, and they will merge the PR after it meets the requirements. Hope this explanation helps. 🙂

This helps clarify things. I had thought this was the case. But I wanted to make sure. Thanks!

jas1005 commented:
This is my first contribution to a repository that I have not owned. What's the next step in the process? I do not currently see my author entry on the open-sauced:main branch.

BekahHW commented:
Hey @jas1005, @adiati98 will merge it in the next time she reviews PRs. Great job!
